tests/Agent/IntegrationTests/UnboundedApplications/BasicMvcApplication/Controllers/MsSqlController.cs

```diff
@@ -10,7 +10,6 @@
 using System.Data.SqlClient;
```

Comment:
I don't think we need this file at all any more, do we?

Reply:
We can remove this controller after the OLEDDB and ODBC tests are migrated. I don't think it is hard to migrate. I just haven't migrated them yet :-)

Reply:
I also realized that I was unsuccessfully migrating the `EnterpriseLibraryMsSqlTests` tests due to some dependencies  errors so we might not yet be able to remove this file.